src/test/scala/com/microsoft/hyperspace/index/rules/RuleUtilsTest.scala

```diff
@@ -147,6 +149,10 @@ class RuleUtilsTest extends HyperspaceRuleTestSuite with SQLHelper {
               assert(
                 indexes.head.getTagValue(plan, IndexLogEntryTags.HYBRIDSCAN_REQUIRED)
                   === expectedHybridScanTag)
+              assert(
+                !hybridScanEnabled || indexes.head
+                  .getTagValue(plan, IndexLogEntryTags.COMMON_SOURCE_SIZE_IN_BYTES) === Some(
+                  expectedCommonSourceBytes))
```

Comment:
Instead of using `var` and capturing the variable as a closure, can we just pass it to `verify` and create a val for each scenario (if necessary)?